lib/membrane_h264_ffmpeg/parser.ex

```diff
@@ -334,6 +324,26 @@ defmodule Membrane.H264.FFmpeg.Parser do
     end
   end
 
+  defp do_handle_stream_format(stream_format, state) do
+    {:ok, %{sps: sps, pps: pps}} =
+      Membrane.H264.FFmpeg.Parser.DecoderConfiguration.parse(
+        stream_format.decoder_configuration_record
+      )
```

Comment:
There's an important conflict to resolve: this should only be parsed if dcr is not `nil`: https://github.com/membraneframework/membrane_h264_ffmpeg_plugin/commit/defcecacc46c3291e3e00a96e2bb050e2ede75a9